JS-111_javascript-loops/working-with-for-loops/code-challenge/terminate-a-loop/script.js

```diff
@@ -0,0 +1,18 @@
+/** JS-111: JavaScript Loops
+ * 
+ * Code challenge: Terminate a Loop
+ */
+let message = "supercalifragilisticexpialidocious";
+message = message.length;
```

Comment:
I'll let this pass. Moving forward, you should name your variables based on what it holds or their purpose.  It would be easier to understand if you assigned `message.length` to a new variable named `messageLength` or anything similar.